### PR TITLE
Fix `podman system connection` panic

### PR DESCRIPTION
**Pull request by jwhonce**
Restored local identity flag

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by baude** [comment]
would you consider a series of simple tests that verify the syntax of whats in the manpages?

**Comment by jwhonce** [comment]
@baude Next commit has tests.  This is just to fix the panic.

File: docs/source/markdown/podman-system-connection.1.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ The user will be prompted for the remote ssh login password or key file pass phr
 
 ## OPTIONS
 
+**--identity**=*path*
+
+Path to ssh identity file. If the identity file has been encrypted, podman prompts the user for the passphrase.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection.1.md:20]:
```suggestion
Path to ssh identity file. If the identity file has been encrypted, Podman prompts the user for the passphrase.
```

File: docs/source/markdown/podman-system-connection.1.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ The user will be prompted for the remote ssh login password or key file pass phr
...
+If no identity file is provided and no user is given, podman defaults to the user running the podman command.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection.1.md:21]:
```suggestion
If no identity file is provided and no user is given, Podman defaults to the user running the Podman command.
```

**Comment by jwhonce** [comment]
GitHub not updating with new commit.

**Pull request closed without merging**